doc/guides/gps_setup.md

```diff
@@ -0,0 +1,57 @@
+---
+module: [kind=guide] gps_setup
+---
+
+# Setting up GPS
+Some quick definitions for this page.
+```
+GPS host: a computer running the GPS program in host mode
+GPS constellation: a group of GPS hosts
+```
+
+Before GPS can be used in a dimention (e.g. the Nether) a GPS host constellation needs to be set up. Traditionally this is done near the max build height for maximum range, but ender modems can beat normal modes in range even from bedrock.
+
+In order to give the best results, a GPS constallation needs atleast four computers. One can theroetically get away with three for most GPS requests, but four changes that to 100% accuracy when the requester is within all of their ranges, provided that they are placed correctly. More than four GPS hosts per constallation is redundent, but it does not harm.
```

Comment:
Typos on line 14:
"atleast" instead of "at least"
"theroetically" instead of "theoretically"
"constallation" instead of "constellation"
"redundent" instead of "redundant"